spec/factories.rb

```diff
@@ -8,4 +8,10 @@
     phone { Faker::PhoneNumber.phone_number }
     password { Faker::Internet.password }
   end
+
+  factory :venue do
```

Comment:
preferred approach is to have each factory in separate file.

Reply:
done